bindings/src/main/java/org/commonjava/rwx/binding/internal/reflect/ReflectionUnbinder.java

```diff
@@ -443,9 +443,6 @@ else if ( binding.getFieldType().isArray() || Collection.class.isAssignableFrom(
                         throw new XmlRpcException(
                                 "Cannot render {} (type: {}) to string. It has no corresponding coercion, and isn't an @ArrayPart or a @StructPart!" );
                     }
-
-                    logger.debug( "Attempting to coerce raw value of: {} using: {}", value, coercion );
-                    value = coercion.toString( value );
```

Comment:
If you remove this, then it looks like the neither the coercion lookup nor the conditional above should be there...

But this change is sort of hard to contextualize. Do you happen to have a way to add a small unit test or something that shows the failure it corrects?